pkg/api/handlers.go

```diff
@@ -190,6 +190,7 @@ func NewQuerierHandler(
 	api := v1.NewAPI(
 		engine,
 		errorTranslateQueryable{queryable}, // Translate errors to errors expected by API.
+		nil,
```

Comment:
Let's just explain why:

```suggestion
		nil, // No remote write support.
```